api/src/main/java/org/openmrs/module/appointmentscheduling/task/AppointmentShedulerSetup.java

```diff
@@ -0,0 +1,49 @@
+package org.openmrs.module.appointmentscheduling.task;
+
+import org.apache.commons.lang.time.DateUtils;
+import org.openmrs.api.context.Context;
+import org.openmrs.scheduler.SchedulerException;
+import org.openmrs.scheduler.SchedulerService;
+import org.openmrs.scheduler.TaskDefinition;
+
+import java.util.Date;
+
+public class AppointmentShedulerSetup {
```

Comment:
slight typo here... Sheduler -> Scheduler

Reply:
@mogoodrich I update the pr as per your reviews. Thanks for the link, it was so much helpful